src/api/worker/WorkerLocator.ts

```diff
@@ -265,26 +274,28 @@ export async function initLocator(worker: WorkerImpl, browserData: BrowserData)
 	)
 	locator.mail = new MailFacade(locator.user, locator.file, locator.cachingEntityClient, locator.crypto, locator.serviceExecutor, locator.blob, fileApp)
 	const nativePushFacade = new NativePushFacadeSendDispatcher(worker)
-	// not needed for admin client
-	if (!isAdminClient()) {
-		locator.calendar = new CalendarFacade(
+	locator.calendar = lazyMemoized(async () => {
```

Comment:
are we just hoping that the admin client never calls this? i guess that's fair

Reply:
that was the hope before as well except it would explode more explicitly